docs/index.html

```diff
@@ -5049,7 +5049,7 @@ <h3 id="why-doesnt-zapier-support-newer-versions-of-nodejs">Why doesn&apos;t Zap
 </div><div class="row">
   <div class="row-height">
     <div class="col-md-5 col-sm-12 col-height  docs-primary">
-      <p>We run your code on AWS Lambda, which only supports a few <a href="https://docs.aws.amazon.com/lambda/latest/dg/programming-model.html">versions</a> of Node (the latest of which is <code>v14</code>. As that updates, so too will we.</p>
+      <p>We run your code on AWS Lambda, which only supports a few <a href="https://docs.aws.amazon.com/lambda/latest/dg/programming-model.html">versions</a> of Node (the latest of which is <code>v16</code>. As that updates, so too will we.</p>
```

Comment:
this is unfortunately not true, since currently Lambda supports up to [Node v18](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html).

```suggestion
      <p>We run your code on AWS Lambda, which only supports a few <a href="https://docs.aws.amazon.com/lambda/latest/dg/programming-model.html">versions</a> of Node.</p>
```

Reply:
Good catch on this one, to be honest I did a search and replace for v14. I've updated the text to just remove a commitment to a version and better explain why we don't support the latest.